docker-compose.yml

```diff
@@ -11,7 +11,7 @@
 version: "3.6"
 services:
   postgres:
-    image: postgres:11-alpine
+    image: postgres:12-alpine
```

Comment:
Surely this is a mistake?

Reply:
I was already wondering where that change went...

Amended to the latest commit